ui/pages/create-account/connect-hardware/index.js

```diff
@@ -277,7 +277,7 @@ class ConnectHardwareForm extends Component {
       });
   };
 
-  onUnlockAccounts = (device, path) => {
+  onUnlockAccounts = async (device, path) => {
```

Comment:
I assume that's fine to make it `async`? I haven't seen any code change regarding this, I saw that we were using it in button handlers (which should already support `async` methods, right?)

Reply:
i make this async is because in line 295, there is `await` call to getHardwareDeviceName()` :)

Reply:
Yes I got that :D but since we've changed the function signature, I would have expect calls to that function to be adapted as well (to explicitly `await` it).

This seems ok since the only references I found for that call are button's event handlers, which allows `async` functions IIRC.